django_socio_grpc/protobuf/tests/protos/test_proto.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
```

Comment:
from which serializer this is generated ?

Reply:
None, written by hand for the tests